docs/NMW-Standard/Location-Encoding.md

```diff
@@ -0,0 +1,196 @@
+
+The locations should be encoded in a hierarchical manner, with each level being more specific than the last.
+
+#### 1.1.
+
+The location code is always presented in plaintext in a human-readable format and will consist of a `nmw:` label, followed by three fields, each separated by a dash (`-`) character.
+
+An example location encoding could therefore be `nmw:1-42-1`.
+
+#### 1.2.
+
+Each field consists of characters from zero to nine and *A* to *Z*. Any alphabetic characters in the location fields should be presented in upper-case form, although the standard is case-insensitive, so lower-case characters should not cause a failure.
+
+#### 1.3.
+
+Each field has a maximum character length that must not be exceeded, but leading zeros may be omitted. This means a field containing `001` is equivalent to a field containing `1`.
+
+The following character length limits apply to the fields:
+
+- Top-level category\\
+    **Length**: 3 characters\\
+    **Max. categories**: 46656
+- Subcategory\\
+    **Length**: 4 characters\\
+    **Max. categories**: 1679616
+- Subsubcategory\\
+    **Length**: 4 characters\\
+    **Max. categories**: 1679616
+
+
+The complete location code can therefore be up to 17 characters long.
+
+#### 1.4
+
+The three location fields in the location code are defined by this standard and no other meanings should be associated with the location codes.
+
+#### 1.5.
+
+Only the first field must be defined in any implementation of the standard. It may be that in some languages or cultures it does not make sense to have a translation at all levels of the hierarchy, but any localisation/regionalisation of the system should have translations for the top-level field.
+
+If any level in the hierarchy is not to be defined this field should be set to `0`. A location code shall be read from left to right, where any fields to the right of a `0` are to be ignored. This means `nmw:1-0-0` and `nmw:1-0-5` have the same meaning, although the latter format is discouraged, because the meaning is not as clear.
+
+_See [§3.1](#31) for a description of what a top-level category of `0` means._
+
+### 2. Unique location identification
+
+The NMW standard allows for additional information to be supplied for locations. This is bound to a [Bluetooth UUID](https://www.bluetooth.com/specifications/assigned-numbers/) for unique identification of locations.
+
+#### 2.1.
+
+UUIDs must be transmitted independently of the location encoding described in [§1](#1).
+
+#### 2.2.
+
+Each location may have a UUID, but it does not need to have one for this standard.
+
+#### 2.3.
+
+Each location's UUID maps to information about that location. This consists of:
+
+| Location attribute | Required? | Description |
+|--------------------|-----------|-------------|
+| NMW location code  | Yes       | The location code according to this standard |
+| Name               | Yes       | A name for the location. This could be, for example the name of a business |
```

Comment:
Could we add one more required field `More Information` or something like that? This would be the more extensive audio snippet that is played to users if they tap the "more info" option. I guess this might be what you meant by the `Description` field, but in that case i suggest we make that field required. What do you think?

Reply:
I've made the description field required so it needs to be present for a UUID mapping to be allowed

Reply:
I think this conversation ca be resolved. Re-open if you think otherwise.